server/apps/research/serializers/article_serializer.py

```
@@ -9,42 +9,66 @@
         model = Article
         fields = [
             'id', 'slug', 'title', 'authors', 'thumb', 'categories', 'summary',
             'acknowledgement', 'content', 'min_read', 'status', 'views',
             'created_at', 'updated_at', 'scheduled_publish_time', 'table_of_contents',
-            'is_sponsored', 'sponsor_color', 'sponsor_text_color'
+            'is_sponsored', 'sponsor_color', 'sponsor_text_color', 'related_articles'
         ]
+        
+    def get_related_articles(self, obj):
+        """Return the related articles for the article."""
+        related_articles = obj.get_related_articles()
```

Ruff (0.8.0)

Local variable `related_articles` is assigned to but never used

Remove assignment to unused variable `related_articles`

(F841)